src/Page/Page.php

```diff
@@ -382,9 +382,27 @@ protected function findParentHook(): string
         if (false !== $pos = strpos($this->getParent(), 'post_type=')) {
             // Parent hook is equivalent to the post type slug value.
             return substr($this->getParent(), $pos + 10);
-        } elseif ('edit.php' === trim($this->getParent(), '\/?&')) {
-            // Parent is the default post post type.
-            return 'posts';
+        } else ('edit.php' === trim($this->getParent(), '\/?&')) {
+            switch (trim($this->getParent(), '\/?&')) {
```

Comment:
The PR seems to be broken. Thus, I suspect that your switch statement is not working as it follows a condition that check against the `edit.php` parent slug... so your switch statement will always return `posts` as the value.

Reply:
I removed the unneeded part after the else, so now it should pass again :).